### DISCUSSION TITLE
大神什么时候更新10.5版本的

### DISCUSSION THREAD

**Discussion by HanQi-rich** [k-skye/gitkraken-chinese#31] [General]
大神什么时候更新10.5版本的

**Comment by YuanXiQWQ** [comment]
抱一丝才看到这里 :( 如果最新文件过时了最好还是写在issue中,那个会显示小数字; 旧版本可以在 [旧版本文件](https://github.com/yk47g/gitkraken-chinese/tree/master/%E6%97%A7%E7%89%88%E6%9C%AC%E6%96%87%E4%BB%B6) 找到
